README.md

```diff
@@ -69,8 +69,8 @@ Prerequisites for building Flink:
 
 * Unix-like environment (we use Linux, Mac OS X, Cygwin, WSL)
 * Git
-* Maven (we require version 3.8.6)
-* Java 8 or 11 (Java 9 or 10 may work)
+* Maven (we require version 4.0.0)
+* Java 17 or 21
```

Comment:
all the CIs are still working with 3.8.6, release activities still require 3.8.6
there was no official drop of 8 and 11

I think both jdk and maven changes like that will require not only doc update but also discussing this with community first via mailing list,  CI updates, release doc updates, may be something else

Reply:
This document is flink 2.0.
According to this [document](https://cwiki.apache.org/confluence/display/FLINK/2.0+Release), Java8 and Java11 have been removed in flink 2.0, Java 17 is the default version and Java 21 is also supported.

Reply:
>According to this [document](https://cwiki.apache.org/confluence/display/FLINK/2.0+Release), Java8 and Java11 have been removed in flink 2.0,

this is planned however not done yet, and this means not only doc update

Reply:
This document is flink 2.0.
According to this [document](https://cwiki.apache.org/confluence/display/FLINK/2.0+Release), Java8 and Java11 have been removed in flink 2.0, Java 17 is the default version and Java 21 is also supported.
The [Maven version](https://maven.apache.org/docs/history.html) was selected as the minimum Java version supported by flink2.0.

Reply:
If you look at code there is still maven 3.8.6
https://github.com/apache/flink/blob/6b08eb15d09d6c8debb609b8b3a2a241ca855155/.mvn/wrapper/maven-wrapper.properties#L19

if you look at ci pipelines there is still jdk8
e.g.
https://github.com/apache/flink/blob/6b08eb15d09d6c8debb609b8b3a2a241ca855155/tools/azure-pipelines/build-apache-repo.yml#L75
https://github.com/apache/flink/blob/6b08eb15d09d6c8debb609b8b3a2a241ca855155/tools/azure-pipelines/build-apache-repo.yml#L108

these are just some examples, in fact there are more

Also if you look at release doc (https://cwiki.apache.org/confluence/display/FLINK/Creating+a+Flink+Release)
there are maven 3.8.6 
and java 8

all this should be discussed/fixed before the doc update

Reply:
OK

Reply:
OK. I canceled the Maven version modification.